02_architecture/index.adoc

```diff
@@ -1,12 +1,12 @@
-== Architecture
+= Architecture
```

Comment:
confluence does not support index.adoc; 
if we want this to be rendered on confluence the structure neends to be:

- 02_architecture.adoc. (as a replacement for index.adoc_
02_architecture folder with subfiles adoc without the index.adoc

Reply:
ndeed the plugin requires a same file.adoc - folder structure

Resources:
https://confluence-publisher.atlassian.net/wiki/spaces/CPD/overview
![Uploading Screenshot 2024-11-04 at 11.27.33.png…]()